extensions/amp-smartlinks/0.1/test/validator-amp-smartlinks.html

```diff
@@ -0,0 +1,37 @@
+<!--
+	Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+			http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS-IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the license.
+-->
+<!--
+	Test Description:
+	Valid amp-smartlinks tag
```

Comment:
Nit: this may eventually test invalid amp-smartlinks in the same file, so perhaps instead `Tests amp-smartlinks tag` (or extension instead of tag).